tests/e2e/testutils.go

```diff
@@ -145,6 +154,27 @@ func DeployWorkload(workload string, kubeconfig string) (string, error) {
 	return "", nil
 }
 
+func RestartCluster(nodeNames []string) error {
```

Comment:
As with K3s, this should probably be RestartServer not RestartCluster. If you're adding new functions, please do add a docstring as well (as you've done for RestartAgent).

Reply:
Thanks Brad, but I did not create the RestartCluster, if I change the name it'll require to change it in any other file that is using the func. I did add the docstring.